src/Model/EditCounter.php

```diff
@@ -1069,4 +1069,18 @@ public function averageEditSize(): float
             return 0;
         }
     }
+
+    /**
+     * Get the sizes of the user's past 5000 edits.
+     * return int[] Array of sizes in bytes.
+     */
+    public function getAllEditSizes(): array
+    {
+        $editSizeData = $this->getEditSizeData();
+        if (isset($editSizeData)) {
```

Comment:
`isset()` is used in the other methods to check for the existence of an array key, because PHP would otherwise throw a warning about undefined array index (though nowadays we should be using the [null coalescing operator](https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op)). Here we don't need to check anything since we want the whole array, and it's guaranteed to be an array thanks to the `array` return type and that we use [strict mode](https://stackoverflow.com/questions/48723637/what-do-strict-types-do-in-php).

So, this method just be `return $this->getEditSizeData()`, effectively an alias. Perhaps instead the callers should use `getEditSizeData()` directly.